src/workload.py

```diff
@@ -185,7 +186,11 @@ def run_bin_command(
         return self.exec(command)
 
     def format_storages(
-        self, uuid: str, internal_user_credentials: dict[str, str] | None = None
+        self,
+        uuid: str,
+        internal_user_credentials: dict[str, str] | None = None,
+        kraft_version: int = KRAFT_VERSION,
+        initial_controllers: str | None = None,
     ) -> None:
```

Comment:
**todo:** The more I think about it, the more I think these kraft specific functions should probably be in their own `managers/kraft.py`.

Reply:
Done! named it `managers/controller.py` to respect other managers' naming convention.